tests/datatypes/test_int.py

```diff
@@ -63,7 +63,6 @@ class BinaryIntOperationTests(BinaryOperationTestCase, TranspileTestCase):
 
     not_implemented = [
         'test_multiply_bytearray',
-        'test_multiply_bytes',
```

Comment:
The test suite failed due to `[gw1] FAILED tests/datatypes/test_int.py::InplaceIntOperationTests::test_multiply_bytes <- tests/utils.py`

You'll need to remove the `'test_multiply_bytes'` at line 87 to get the suite to pass